lab/lab1/part3-data-transformation.html

```diff
@@ -99,12 +107,33 @@
     ===================== */
 
 
+    dataFiltered.forEach(function(element){
+    var makeAmarker =  L.marker(element);
+    makeAmarker.addTo(map)
+    });
 
     /* =====================
 
     End code to add markers to map
 
     ===================== */
+    //change code to account for the
+    //dirty data
+
+    var dataFiltered = [];
+
+    for (var i = 0; i < bikeArrayDirty.length; i = i+1) {
+    //if (bikeArrayClean[i][3] > 20) {
+    var add_loc = [];
+    add_loc.push(bikeArrayClean[i][1]);
+    add_loc.push(bikeArrayClean[i][0]);
```

Comment:
Looks like you used the `BikeArrayDirty` to count, but not to actually add data to the map:

![image](https://user-images.githubusercontent.com/8103418/52545870-e60e2680-2d88-11e9-8f72-1c0157075aac.png)



![image](https://user-images.githubusercontent.com/8103418/52545865-ddb5eb80-2d88-11e9-8b8e-7035beeb0589.png)


![image](https://user-images.githubusercontent.com/8103418/52545861-d68edd80-2d88-11e9-857d-5d11656c1ef9.png)

The last image is what happened when I switched it to dirty data...so you were very close...